controlplane/kubeadm/api/v1alpha4/kubeadmcontrolplane_default.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha4
+
+import (
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func (s *KubeadmControlPlaneSpec) setDefaults(namespace string) {
```

Comment:
Let's hear also other comments, but for consistency with the other webhooks I would like those function to be implemented in kcp_webhook.go as func defaultKubeadmControlPlaneSpec(s *KubeadmControlPlaneSpec)

Reply:
there have been no further comments.
I will refactor this.

Reply:
also @fabriziopandini just to be sure you dont mean to create a new new file called `kcp_webhook.go`, instead you mean to move these defaulting (also validation?) functions to [kubeadm_control_plnae_webhook.go](https://github.com/kubernetes-sigs/cluster-api/blob/master/controlplane/kubeadm/api/v1alpha4/kubeadm_control_plane_webhook.go), correct?

Reply:
@ykakarap Exactly